src/indigo/components/AlertBlock.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {
+    Alert
+} from 'react-bootstrap';
+
+const AlertBlock = React.createClass({
+    propTypes: {
+        title: React.PropTypes.bool.isRequired,
+        style: React.PropTypes.bool.isRequired,
+        children: React.PropTypes.any
```

Comment:
@janmichek ak mozes este polad tieto props. title a style nemaju byt bool. A children by som dal required, bez toho to asi nema vyznam, nie?

Reply:
good point, prej bool xDD